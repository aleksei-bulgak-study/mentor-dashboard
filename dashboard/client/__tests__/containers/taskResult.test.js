import React from 'react';
import { shallow } from 'enzyme';
import TaskResultContainer from '../../src/containers/taskResult';

describe('TaskResultContainer', () => {
  it('test snapshot does not changed', () => {
    // given
    const props = {
      task: {
        name: 'test',
        status: 'Checked',
      },
      results: {
        test: {
          mark: 10,
          pr: 'http://example.com',
        },
      },
    };
    // when
    const wrapper = shallow(<TaskResultContainer {...props} />);
    // then
    expect(wrapper).toMatchSnapshot();
  });
});
