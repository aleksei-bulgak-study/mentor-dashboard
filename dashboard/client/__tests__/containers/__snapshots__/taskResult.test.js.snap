// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskResultContainer test snapshot does not changed 1`] = `
<TaskResultColumnContainer
  result={
    Object {
      "mark": 10,
      "pr": "http://example.com",
    }
  }
  status="Checked"
/>
`;
